### PR TITLE
Permit running on remote servers

### PR DESCRIPTION
**Pull request by miguel-guedes**
Tests can now be run on remote servers (Selenium server or Sauce Labs) or locally.  Further work required to enable remote tests to connect to an existing Dossier stack instance and reset state to a pre-determined point.

### CODE REVIEW

File: src/browser/extensions/chrome-testing.pem [hunk 1]
```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```
Comment (BurntSushi) [inline src/browser/extensions/chrome-testing.pem]: @miguel-guedes What is this a private key for? Is it OK to be in a public repo?

Comment (miguel-guedes) [inline src/browser/extensions/chrome-testing.pem]: Absolutely fine, yes. It is only being used to package the extension correctly to ensure that the Chrome driver accepts it. This involves invoking the actual Chrome/ium browser so it emits a .crx file and requires a private key file.  As I say, the actual key file is only being used for testing purposes.

Comment (BurntSushi) [inline src/browser/extensions/chrome-testing.pem]: Excellent.

**Comment by BurntSushi** [comment]
Sweet! Thanks!

**Pull request merged**